.github/workflows/tests_cuda.yml

```diff
@@ -21,6 +21,7 @@ jobs:
     - name: Install dependencies
       shell: bash -l {0}
       run: |
+        apt-get install git
```

Comment:
I have not seen the need to do this before but perhaps is caused by line 22.

Reply:
we need git in case someone is installing the package from source. See: https://github.com/kornia/kornia/blob/master/setup.py#L67

In the cpu workflow it's not needed because we create the environment installing the dependencies by hand - but in the case of cuda we reuse pytorch docker image which apparently doesn't include git.


Not sure if that's what you meant :)

Reply:
I meant that `git` was always present in GitHub Actions.

Reply:
ah! mmm, then something is wrong from our side

Reply:
@cclauss I checked in local as well and the pytorch image we use doesn't include git. Was reading the dockerfile but quite not sure were and why they remove it: https://github.com/pytorch/pytorch/blob/master/Dockerfile